pages/api/models/selfCheckGroup.ts

```diff
@@ -0,0 +1,26 @@
+import { model, Schema } from "mongoose";
+
+interface SelfCheckGroupInterface {
+    _id: Schema.Types.ObjectId;
+    questionIDs: [Schema.Types.ObjectId];
+}
+
+const SelfCheckGroupSchema = new Schema<SelfCheckGroupInterface>(
+    {
+        questionIDs: {
+            type: [{ type: Schema.Types.ObjectId, ref: "Questions" }],
+            default: [],
+        },
+    },
+    {
+        timestamps: {
```

Comment:
lets just use `{timestamps: true}`

Reply:
This just gets the createdAt and updatedAt fields (which is prob the only ones we need)